### ISSUE TITLE
What is this?

### ISSUE THREAD

**Issue by xameeramir** [PaoloRotolo/android-studio#18]
@PaoloRotolo I saw this project for the first time and I feel that it automates installation of Android studio which is too much command-line-hard-work.
Did I got it right? How can I explain this to a [child](http://www.goodreads.com/quotes/19421-if-you-can-t-explain-it-to-a-six-year-old)? I would suggest you to make your product description understandable in layman terms.

**Comment by PaoloRotolo** [comment]
Hi @xameeramir.
Sure, you can read the "How it works" section on the official website: https://paolorotolo.github.io/android-studio/

**Comment by xameeramir** [comment]
@PaoloRotolo The idea is really great! Can you put "how it works" section in **front of your website visitors eyes**. Suggesting from UX point of view.
